.github/workflows/release.yml

```diff
@@ -0,0 +1,114 @@
+name: Release
+on:
+  push:
+    tags:
+      - v*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
```

Comment:
Why should warnings fail building the zips?

Reply:
It is probably best to not ship with a release with warnings? 🤔

Reply:
That should already fail the main CI workflow.

Reply:
Right, but those workflows are mostly unrelated right? And a failed build should then result in no artifacts as well, right?